apps/settings/manifest.webapp

```diff
@@ -44,7 +44,8 @@
         "target": "device"
        },
       "href": "index.html#activity-configure",
-      "disposition": "inline"
+      "disposition": "inline",
+      "returnValue": true
```

Comment:
Hey Andre, Fred, I just wonder if it could be more right to use window.close instead of postResult ? Because the "configure" activity is not really returning anything...